app/src/main/java/com/codesoom/assignment/TodoHttpHandler.java

```diff
@@ -0,0 +1,115 @@
+package com.codesoom.assignment;
+
+import com.codesoom.assignment.models.Task;
+import com.codesoom.assignment.models.Title;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.sun.net.httpserver.HttpExchange;
+import com.sun.net.httpserver.HttpHandler;
+
+import java.io.*;
+import java.net.URI;
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.stream.Collectors;
+
+public class TodoHttpHandler implements HttpHandler {
+    private ObjectMapper objectMapper = new ObjectMapper();
+    private Map<Long, Task> taskMap = new ConcurrentHashMap<>();
+
+    private static final String GET = "GET";
+    private static final String POST = "POST";
+    private static final String PUT = "PUT";
+    private static final String PATCH = "PATCH";
+    private static final String DELETE = "DELETE";
+    private static final String BASIC_URI = "/tasks";
+
+    @Override
+    public void handle(HttpExchange exchange) throws IOException {
+        String method = exchange.getRequestMethod();
+        String path = exchange.getRequestURI().getPath();
+        String content = "";
+        InputStream inputStream = exchange.getRequestBody();
+        String body = new BufferedReader(new InputStreamReader(inputStream))
+                .lines()
+                .collect(Collectors.joining("\n"));
+
+        // TODO: 나중에 mehotd들 enum으로 변경해보기
+        if (method.equals(GET)) {
```

Comment:
이렇게 바꾸면 어떤 장점이 있을까요? 어떤 문제를 해결하게 될까요?
```suggestion
        if (GET.equals(method)) {
```

Reply:
만약 변수.equals("문자열")일 경우에, 변수가 null일 경우 NullPointerException이 발생하는 문제가 있습니다. 그래서 "문자열".equals(변수)를 하면 그 문제를 해결 할 수 있습니다. 찾아본 바에 의하면, 오히려 NullPointerException 문제를 터트릴 수 있을 때 확실히 터트리는 것(?)을 선호해서 전자를 쓰는 사람도 있다고 하지만, 저는 후자가 더 나은 것 같습니다!

Reply:
잘 조사하셨습니다 :-)